recycler/src/main/java/com/elpassion/android/commons/recycler/items/ItemAdapter.kt

```diff
@@ -1,4 +1,4 @@
-package com.elpassion.android.commons.recycler
+package com.elpassion.android.commons.recycler.items
```

Comment:
I think of adapter as an algorithm. So we usually need one instance of adapter for one type of items. Not new adapter for every item instance. For example if our RecyclerView is just a list of 100 emails. We only need one adapter that can adapt any email <-> view/holder. It feels strange to have 100 separate adapters...